lib/resty/t1k/request.lua

```diff
@@ -267,6 +267,7 @@ local function do_socket(opts, header_buf, payload_buf)
 end
 
 function _M.do_request(opts)
+    -- why do you set this?
```

Comment:
previously added for statistical purposes, so does the `ngx.ctx.t1k_request_success` and `ngx.ctx.t1k_request_time`. These data can be collected in the `log_by_lua_*` phase.

These statistical data can be removed.